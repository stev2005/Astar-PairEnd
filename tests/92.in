TTTCCCCTCGAAGGAGCGTAAGTCCACCCCGCAGGCTTAAGAAATCGATGATATACACGCGCGGTATGAAG
15
ATGTAATTGTGGTGA
TGAGTTAG
AATATCAGTAGATCGTTCGCCCATCGTATCATGTTCCTTATCATGTGCAAGCCGAG
TTACTTTGGATCTACAAAGTGTTGCATTCC
AAACTTAGCACTTCATGACCG
CACCTCGACTCGAACCGTCAGCCGATGGC
GGAATTCGTGCAACCGGTCATTCCCCATTCTGTTGTGTCCC
GGTTAGGCGGA
TCATAACTGAGA
ATACAGTGGGGACTGTTCGACCA
GTTTCTGCGTTTCGCTCATGTCCTGTTAA
CTTCGCAAGTTCGCATACTATTCAGATCATCCGCGAGTCAGAGACGTGTT
GGTGATCCTATCTATTGGAGACAAGGAACTGTCGCCCGGCTCGGCCCTAGGATGCCCCG
AAGCGTGAGACCGTTTAATAGCCCTGTATCTTCCAAAGAGGCTAAGAATTCCAG
TAGATTTGAAGAAAATGCGTCCTGAGTCGCACCGCACAGGAAGCATA
