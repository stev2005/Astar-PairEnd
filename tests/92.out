6
3
27
15
10
12
22
4
5
10
14
23
28
24
20
